web-app/src/services/holidayService.js

```diff
@@ -24,10 +24,13 @@ export const approveHoliday = eventId => {
   return axios.put('/holidays/approveHoliday', { eventId });
 };
 
-export const rejectHoliday = eventId => {
+export const rejectHoliday = (eventId, message) => {
+  if (message) {
```

Comment:
Thinking a reject object would be a lot cleaner here.

`{
  eventId: 1,
  message: 'rejection message here'
}`

Maintainability plus a cleaner pure service here. No logic should exist in the services, it takes the parameters and hits the endpoint. @eoinmcafee would it be ok your side, to take an object instead of two strings and pull the values out of the object.

Reply:
Inspecting this method being called, there is reference to rejectHoliday elsewhere but it is a different function completely so the only place that calls rejectHoliday to hit that endpoint is the modal which always requires a rejection message so I have removed the logic and sending the message at all times as it should always be being passed in.

Reply:
Fair enough. As long as the service is clean.